canonical-web-and-design/maas-ui
# Action button progress

## Done
- Use the action button to show progress in the take action menu.

## QA
- In the machine list use the take action menu to perform various actions.
- While the machines are being processed the button should turn into a spinner.
- Perform an action that results in an error, you should see the error in the header but not a duplicate error in the machine list.

## Fixes
Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/1899.
Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1914.
